airflow/assets/__init__.py

```diff
@@ -24,6 +24,7 @@
 from typing import TYPE_CHECKING, Any, Callable, ClassVar, Iterable, Iterator, cast, overload
 
 import attr
+import attrs
```

Comment:
Let’s unify these, they are the same lib and we should only import from one of them at a time. (Does not really matter which; `attrs` is probably better if we only use the type annotation style.)

https://www.attrs.org/en/stable/names.html

Reply:
Yep, I was confused a bit and decided to check it later and forgot... thanks for reminding me. will update it.